action/protocol/poll/consortium.go

```diff
@@ -149,7 +152,15 @@ func (cc *consortiumCommittee) CreateGenesisStates(ctx context.Context, sm proto
 	}
 	cc.contract = receipt.ContractAddress
 
-	r := getContractReaderForGenesisStates(ctx, sm, cc.getBlockHash)
+	ctx = evm.WithHelperCtx(ctx, evm.HelperContext{
+		GetBlockHash: cc.getBlockHash,
+		GetBlockTime: cc.getBlockTime,
+		DepositGasFunc: func(context.Context, protocol.StateManager, address.Address, *big.Int, *big.Int) (*action.TransactionLog, error) {
+			return nil, nil
+		},
```

Comment:
is this the real depositGasFunc used? i mean, do nothing b/c it is a reader?

Reply:
we could keep it as `nil` as original